kwok/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.k8s.io/kwok/cluster:v0.3.0-k8s.v1.27.3
+
+COPY user-signups.sh ./user-signups.sh 
+
+RUN chmod +x ./user-signups.sh
+
+COPY entrypoint.sh ./entrypoint.sh
+
+RUN wget https://raw.githubusercontent.com/codeready-toolchain/host-operator/master/config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml
+
+RUN chmod +x ./user-signups.sh ./entrypoint.sh ./toolchain.dev.openshift.com_usersignups.yaml
```

Comment:
you can chmod with the copy command, no need to create an extra image layer for that.

Reply:
Done